### ISSUE TITLE
Publish v1.0.4

### ISSUE THREAD

**Issue by nathan7** [iarna/are-we-there-yet#3]
This is causing breakage on 0.8:
```
$ npm install
util.js:538
  ctor.prototype = Object.create(superCtor.prototype, {
                                          ^
TypeError: Cannot read property 'prototype' of undefined
    at Object.exports.inherits (util.js:538:43)
    at Object.<anonymous> (/home/travis/.nvm/v0.8.28/lib/node_modules/npm/node_modules/npmlog/node_modules/are-we-there-yet/index.js:115:6)
    at Module._compile (module.js:449:26)
    at Object.Module._extensions..js (module.js:467:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:362:17)
    at require (module.js:378:17)
    at Object.<anonymous> (/home/travis/.nvm/v0.8.28/lib/node_modules/npm/node_modules/npmlog/log.js:2:16)
    at Module._compile (module.js:449:26)
```

**Comment by nathan7** [comment]
Ah, seems this was fixed in #2, but not yet published. Can we publish this so we can update the version in `npm@2`?

**Comment by iarna** [comment]
nathan7: That was landed in 1f380f6 and published in `npm@2.7.6`, which is currently `npm@next`. It will be pushed to `npm@latest` this week per the usual release schedule.
